homeassistant/components/circuit/notify.py

```diff
@@ -0,0 +1,40 @@
+"""Unify Circuit platform for notify component."""
+import logging
+
+import attr
+from circuit_webhook import Circuit
+
+from homeassistant.components.notify import ATTR_TARGET, BaseNotificationService
+from homeassistant.const import CONF_URL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the Unify Circuit notification service."""
+    if discovery_info is None:
+        return None
+
+    return CircuitNotificationService(hass, discovery_info)
+
+
+@attr.s
+class CircuitNotificationService(BaseNotificationService):
+    """Implement the notification service for Unify Circuit."""
+
+    hass = attr.ib()
```

Comment:
We don't need `hass`.

Reply:
Ok, I rewrote the code. Will commit now.